addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRuleUnitTest.java

```diff
@@ -207,6 +207,32 @@ void shouldAlertOnWildcardFrameAncestorsDirective() {
         assertThat(alertsRaised.get(0).getAlertRef(), equalTo("10055-4"));
     }
 
+    @Test
+    void shouldAlertOnWildcardConnectSourceDirective() {
+        // Given
+        HttpMessage msg =
+                createHttpMessage("connect-src *; default-src 'self'; form-action 'none'");
```

Comment:
hmm wait wait… should not we include `frame-ancestors`  here so it is not included in the error below?

This test would otherwise test both changes so uhm…

Reply:
Sure that'd be more precise.